libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/TriggerConditions/OnActivity.cs

```diff
@@ -15,7 +15,7 @@ namespace Microsoft.Bot.Builder.Dialogs.Adaptive.Conditions
     public class OnActivity : OnDialogEvent
     {
         [JsonProperty("$kind")]
-        public const string DeclarativeType = "Microsoft.OnActivity";
+        public new const string DeclarativeType = "Microsoft.OnActivity";
```

Comment:
Ha

Reply:
virtual get-only properties might have been more intuitive here, but that train may have left the station. There are downsides to properties too, it's a trade off I guess.